Common/src/CConfig.cpp

```diff
@@ -6824,7 +6872,23 @@ void CConfig::SetOutput(SU2_COMPONENT val_software, unsigned short val_izone) {
 
   if (val_software == SU2_COMPONENT::SU2_CFD) {
 
-    cout << "Writing solution files every " << VolumeWrtFreq <<" iterations."<< endl;
+    cout << "File writing frequency: " << endl;
+    PrintingToolbox::CTablePrinter FileFreqTable(&std::cout);
+    FileFreqTable.AddColumn("File",         25);
+    FileFreqTable.AddColumn("Frequency",    10);
```

Comment:
Looks good but no "pretty formatting".
```suggestion
    FileFreqTable.AddColumn("File", 25);
    FileFreqTable.AddColumn("Frequency", 10);
```